packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-node.vue

```diff
@@ -30,17 +32,41 @@ const isModelInputConnected = computed(() => modelInput?.status === WorkflowPort
 
 watch(
 	() => props.node.inputs,
-	(inputs) => {
+	async (inputs) => {
+		const state = cloneDeep(props.node.state);
 		const modelId = inputs.find((input) => input.type === 'modelId')?.value?.[0];
 		if (!modelId) return;
 
-		const state = cloneDeep(props.node.state);
-		// Reset previous model cache
-		state.interventionPolicy = {
-			modelId,
-			interventions: [blankIntervention]
-		};
-		emit('update-state', state);
+		let policies = await getInterventionPoliciesForModel(modelId);
+		if (!policies[0]?.id) {
+			const interventionPolicy: InterventionPolicy = {
+				modelId,
+				name: 'New Intervention Policy',
+				interventions: [blankIntervention]
+			};
+
+			await createInterventionPolicy(interventionPolicy);
+			policies = await getInterventionPoliciesForModel(modelId);
+		}
+
+		if (policies[0]?.id) {
+			const policy = policies[0];
+			state.interventionPolicy = policy;
+			emit('update-state', state);
+			emit('append-output', {
+				type: InterventionPolicyOperation.outputs[0].type,
+				label: policy.name,
+				value: policy.id,
+				isSelected: false,
+				state
+			});
+		} else {
+			state.interventionPolicy = {
+				modelId,
+				interventions: [blankIntervention]
+			};
+			emit('update-state', state);
+		}
```

Comment:
Why are we changing this watcher and creating a "default" intervention policy?

Reply:
I thought a default policy would make sense, but a blank one doesnt really 😬.  I've removed this